showmore_dyn/showmore_dyn.js

```diff
@@ -0,0 +1,81 @@
+$(document).ready(function(){
+        handleNewWallItemBodies();
+
+        var mutationObserver = new MutationObserver(function(mutations) {
+                handleNewWallItemBodies();
+        });
+        mutationObserver.observe($("#content")[0], { attributes: false, characterData: false, childList: true, subtree: true, attributeOldValue: false, characterDataOldValue: false });
+});
+
+function handleNewWallItemBodies() {
+        $('.wall-item-body:not(.showmore-done)').each(function(i, el) {
+                $(el).addClass('showmore-done');
+                if ($(el).has('button.content-filter-button').length > 0) {
+                        $(el).removeClass('limitable');
+                        return;
+                }
+
+                var itemId = $(el).attr('id');
+                addHeightToggleHandler(itemId);
```

Comment:
So, you have a full jQuery object with an id, you extract the id, pass it to `addHeightToggleHandler` where you then query the full jQuery object. I feel like we could remove a couple steps and just pass the full jQuery object directly.